src/progressively.js

```diff
@@ -1,158 +1,234 @@
 /*!
```

Comment:
You must preserve the code style of the project. Progressively follows [standardjs](http://standardjs.com/) so do not use semicolons and tabs. Just run `npm run lint` to see what you should change.